core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -570,6 +571,11 @@ class ChecksumMessageFormatter extends MessageFormatter {
   }
 
   def writeTo(consumerRecord: ConsumerRecord[Array[Byte], Array[Byte]], output: PrintStream) {
-    output.println(topicStr + "checksum:" + consumerRecord.checksum)
+    val checksum = DefaultRecord.computePartialChecksum(
+      consumerRecord.timestamp(),
+      consumerRecord.serializedKeySize(),
+      consumerRecord.serializedValueSize()
+    )
+    output.println(s"${topicStr}checksum:$checksum")
```

Comment:
I don't think this one makes sense. We should instead consider using https://github.com/ghik/silencer. There's a JIRA about this.